src/schemas/21A/schema.js

```diff
@@ -0,0 +1,513 @@
+    },
+    employmentStatusExplanation: {
+      type: 'string',
+    },
+    businessAddress: {
+      type: 'object',
+      properties: {
+        addressType: {
+          type: 'boolean',
+        },
+        line1: {
+          type: 'string',
+        },
+        line2: {
+          type: 'string',
+        },
+        city: {
+          type: 'string',
+        },
+        state: {
+          type: 'string',
+        },
+        postalCode: {
+          type: 'string',
+        },
+        country: {
+          type: 'string',
+          enum: countries.map(x => x.value),
+        },
+      },
+    },
+    birthdate: {
+      $ref: '#/definitions/date',
+    },
+    birthCity: {
+      type: 'string',
+    },
+    birthState: {
+      type: 'string',
+    },
+    birthCountry: {
+      type: 'string',
+      enum: countries.map(x => x.value),
+    },
+    militaryServices: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          serviceBranch: {
+            type: 'object',
+            properties: {
+              name: {
+                type: 'string',
+              },
+            },
+          },
+          dischargeType: {
+            type: 'object',
+            properties: {
+              name: {
+                type: 'string',
+              },
+            },
+          },
+          dischargeTypeExplanation: {
+            type: 'string',
+          },
+          entryDate: {
+            $ref: '#/definitions/date',
+          },
+          dischargeDate: {
+            $ref: '#/definitions/date',
+          },
+        },
+      },
+    },
+    employment: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          employerAddress: {
+            type: 'object',
+            properties: {
+              line1: {
+                type: 'string',
+              },
+              line2: {
+                type: 'string',
+              },
+              city: {
+                type: 'string',
+              },
+              state: {
+                type: 'string',
+              },
+              postalCode: {
+                type: 'string',
+              },
+              country: {
+                type: 'string',
+                enum: countries.map(x => x.value),
+              },
+            },
+          },
+          phoneNumber: {
+            $ref: '#/definitions/phone',
+          },
+          phoneExtension: {
+            type: 'string',
+          },
+          positionTitle: {
+            type: 'string',
+          },
+          startDate: {
+            $ref: '#/definitions/date',
+          },
+          endDate: {
+            $ref: '#/definitions/date',
+          },
+          supervisorName: {
+            type: 'string',
+          },
+          supervisorEmail: {
+            $ref: '#/definitions/email',
+          },
+        },
+      },
+    },
+    financialPlaning: {
```

Comment:
this has a typo, `financialPlanning`

I also believe that all of these keys (financialPlanning, homeNursingCare, medicalServices, consultingService, advertisingToVeterans) should be booleans instead of strings